src/main/java/com/google/alpollo/model/SongInfo.java

```diff
@@ -0,0 +1,38 @@
+package com.google.alpollo.model;
+
+import java.util.List;
+import com.google.alpollo.Song;
+import com.google.alpollo.model.SongEntity;
+import com.googlecode.objectify.annotation.Index;
+import com.googlecode.objectify.annotation.Entity;
+
+/**
+ * Class used for storing data related to the Natural Language API analysis.
+ * Extending the Song class, it provides all the info related to a specific song.
+ * 
+ * We extend from Song because we do not always need the entire song data. Sasha mostly
+ * needs to work with the "superficial" data (artist name, song name etc.) while Budi has to 
+ * work with the analysis results.
+ */
+@Entity
+public class SongInfo extends Song {
+  /** Score tells us the overall positivty of the song, ranging from -1.0 to 1.0. */
+  @Index private float score;
+  /** Magnitude tells us how strong the main sentiment of the song is, ranging from 0.0 to +inf. */
+  @Index private float magnitude;
+  /** TopSalientEntities is a list containing the 10 most important words, given the song context. */
+  @Index private List<SongEntity> topSalientEntities;
+  @Index private String lyrics;
+  /** After analyzing the entities, we'll have a list of recommended YouTube links. */
```

Comment:
There's no need to explain in documentation how we made this recommendations, it's just smth like 
`List of recommended YouTube video IDs, based on song lyrics/entity analysis.`